wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -6,39 +6,54 @@ ul.sequence {
 li.sequence-member {
     @include clearfix;
     position: relative;
-    border: 1px solid transparent;
+    border: solid $color-grey-4;
     border-width: 1px 0;
 
-    &:hover {
-        background-color: $color-input-focus;
-        border-color: darken($color-input-focus, 10%);
-
-        .sequence-member-inner {
-            > .struct-block > label,
-            > .char_field > label,
-            .sequence > label {
-                opacity: 1;
-            }
-        }
+    &:first-child {
+        border-top: 0;
     }
 
-    &:hover .object-help {
+    // Note this inherits .oject-help from page-editor.scss
+    .object-help {
+        display: block;
+        float: none;
+        font-style: italic;
+        margin: 0;
+        max-width: 650px;
         opacity: 1;
+        padding-left: 0;
+        position: relative;
+        width: 100%;
+
+        &::before {
+            display: none;
+        }
+        @media screen and (max-width: $breakpoint-mobile) {
+            margin: -20px 0 30px;
+            padding-left: 0;
+        }
     }
 
     .struct-block .fields {
-        @include column(10);
+        width: 100%;
         padding-left: 0;
         padding-right: 0;
     }
 
+    .struct-block label {
```

Comment:
This is quite a far-reaching selector, considering pretty much any field can be inside a Streamfield. I'd suggest going for something more precise if possible (eg. `.sequence-type-stream .fields > li > label`, more specific but targets less potential elements).

Reply:
Good point. It's tricky as lines 6 - 153 could be wrapped in a `.sequence-type-stream` class if it weren't for my fear that some of these declarations are being used by lists (since `sequence_member.html` is extended by both `list_member.html` and `stream_member.html`). For lack of better ideas I've followed your suggestion by pulling it out of the `li.sequence-member` selector and on its own.